js/calendar.js

```diff
@@ -7,9 +7,21 @@
  */
 "use strict";
 
-Date.prototype.getWeek = function() {
-	var onejan = new Date(this.getFullYear(), 0, 1);
-	return Math.ceil((((this.getTime() - onejan.getTime()) / 86400000) + onejan.getDay() + 1) / 7);
+Date.prototype.getWeek = function(iso8601) {
```

Comment:
Why do you path `iso8601` into function like argument? Why cannot we use `this.options.week_numbers_iso_8601`?